docs/user/geojson.rst

```diff
@@ -0,0 +1,223 @@
+JSON and GeoJSON
+================
+
+GeoMesa provides built-in integration with JSON and GeoJSON. JSON can be stored as an attribute in a simple feature
+and queried using JSONPath through CQL. For users who are not interested in GeoTools, GeoMesa provides a GeoJSON
```

Comment:
Can we add a link to JSONPath for those who may not be familiar with its syntax? E.g. http://goessner.net/articles/JsonPath/